packages/mask/src/plugins/Savings/constants/venus.ts

```diff
@@ -0,0 +1,391 @@
+    ],
+    [
+        createERC20Tokens('LTC_ADDRESS', 'LTC', 'LTC', 6)[ChainId.BSC],
+        createERC20Tokens('vLTC_ADDRESS', 'Venus LTC', 'vLTC', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('XRP_ADDRESS', 'XRP', 'XRP', 6)[ChainId.BSC],
+        createERC20Tokens('vXRP_ADDRESS', 'Venus XRP', 'vXRP', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('BCH_ADDRESS', 'BCH', 'BCH', 6)[ChainId.BSC],
+        createERC20Tokens('vBCH_ADDRESS', 'Venus BCH', 'vBCH', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('DOT_ADDRESS', 'DOT', 'DOT', 6)[ChainId.BSC],
+        createERC20Tokens('vDOT_ADDRESS', 'Venus DOT', 'vDOT', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('LINK_ADDRESS', 'LINK', 'LINK', 6)[ChainId.BSC],
+        createERC20Tokens('vLINK_ADDRESS', 'Venus LINK', 'vLINK', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('DAI_ADDRESS', 'DAI', 'DAI', 6)[ChainId.BSC],
+        createERC20Tokens('vDAI_ADDRESS', 'Venus DAI', 'vDAI', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('FIL_ADDRESS', 'FIL', 'FIL', 6)[ChainId.BSC],
+        createERC20Tokens('vFIL_ADDRESS', 'Venus FIL', 'vFIL', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('BETH_ADDRESS', 'BETH', 'BETH', 6)[ChainId.BSC],
+        createERC20Tokens('vBETH_ADDRESS', 'Venus BETH', 'vBETH', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('ADA_ADDRESS', 'ADA', 'ADA', 6)[ChainId.BSC],
+        createERC20Tokens('vADA_ADDRESS', 'Venus ADA', 'vADA', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('DOGE_ADDRESS', 'DOGE', 'DOGE', 6)[ChainId.BSC],
+        createERC20Tokens('vDOGE_ADDRESS', 'Venus DOGE', 'vDOGE', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('CAKE_ADDRESS', 'CAKE', 'CAKE', 6)[ChainId.BSC],
+        createERC20Tokens('vCAKE_ADDRESS', 'Venus CAKE', 'vCAKE', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('MATIC_ADDRESS', 'MATIC', 'MATIC', 6)[ChainId.BSC],
+        createERC20Tokens('vMATIC_ADDRESS', 'Venus MATIC', 'vMATIC', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('AAVE_ADDRESS', 'AAVE', 'AAVE', 6)[ChainId.BSC],
+        createERC20Tokens('vAAVE_ADDRESS', 'Venus AAVE', 'vAAVE', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('TUSD_ADDRESS', 'TUSD', 'TUSD', 6)[ChainId.BSC],
+        createERC20Tokens('vTUSD_ADDRESS', 'Venus TUSD', 'vTUSD', vtokenDecimals)[ChainId.BSC],
+    ],
+    [
+        createERC20Tokens('TRX_ADDRESS', 'TRX', 'TRX', 6)[ChainId.BSC],
+        createERC20Tokens('vTRX_ADDRESS', 'Venus TRX', 'vTRX', vtokenDecimals)[ChainId.BSC],
+    ],
+]
+
+export const CONTRACT_VBEP_ADDRESS = [
+    {
+        id: 'sxp',
+        symbol: 'vSXP',
+        address: getTokenConstants(ChainId.BSC).vSXP_ADDRESS,
```

Comment:
extract the `getTokenConstants(ChainId.BSC)` as variable?

Reply:
I'll change it to how we use it in the project.